ember-toucan-core/src/components/form/checkbox-field.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  class="flex flex-col"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Control
+      class="rounded-sm p-1 {{if @error 'shadow-error-outline'}}"
+      data-control
+    >
+
+      <label class="flex flex-row space-x-3" for={{field.id}}>
+        <Form::Controls::Checkbox
+          aria-invalid={{if @error "true"}}
+          aria-describedby="{{if @error field.errorId}}"
```

Comment:
Since the label text (span below) + hint are wrapped inside the label, we don't need to add them to `aria-describedby` here for screenreaders 🎉